### ISSUE TITLE
[TESTING]: OMB Info - Unit tests MUST run axe checks with the modal open for better code coverage

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#9816]
## [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

The OMB Info component includes a modal window of legal text. The unit tests don't run an axe check on this modal. The component depends on this modal, so it must include an axe check with the modal open. This looks like it can be passed as a prop easily in the `OMBInfo.unit.spec.jsx` file.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] Unit test file includes an axe check with the modal window open
- [ ] Axe checks return 0 violations

## Environment

* https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/ombinfo/

**Comment by 1Copenut** [comment]
Issue moved to [department-of-veterans-affairs/vets-design-system-documentation #224](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/224) via [**ZenHub**](https://www.zenhub.com/)
